### ISSUE TITLE
Text area field character limit is not enforced

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#608]
**Description**

The character limit on text fields doesn't appear to be enforced - once the character counter reaches zero, I am able to continue entering content and submit the form.

**Steps to reproduce**
1. Create text area field with 254 character limit
2. Enter 255 or more characters

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes?
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.21.1
- Craft version: Pro 3.7.16
- Multi-site: No

**Comment by engram-design** [comment]
We use JS to limit text field content with to words and characters. Can you confirm if the JS is being output on the page? It seems like it is, if the counter value is changing as you type. What browser and version are you using? I've tested in most modern browsers and prevents typing.

As per the [code](https://github.com/verbb/formie/blob/craft-3/src/web/assets/frontend/src/js/fields/text-limit.js) we act on the `keydown` event, so there shouldn't be any oddities with that...

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.21.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.22](https://github.com/verbb/formie/releases/tag/1.4.22)
